### ISSUE TITLE
Remove deprecated extensions

### ISSUE THREAD

**Issue by hbrands** [glazedlists/glazedlists#666]
With GL 1.12 we deprecated the following extensions:
- io
- ktable
- nachocalendar
- jgoodiesforms

These should be removed from the official distribution and become unsupported with the next major release.
I propose to create a separate project/repo "glazedlists-unsupported" und move those extensions there, ideally with a working gradle build and a dependency on the last working Glazed Lists version at that time. So people would know it's unsupported but still accessible, clonable and buildable.

**Comment by fab1an** [comment]
Good idea, but for sake of saving work, I would wait until someone actually comes along who needs this. Maybe ask on the mailing list.
